src/mokka/src/test/java/pl/hycom/mokka/service/openapi/OpenAPIToStubMappingListWrapperConverterUnitTest.java

```diff
@@ -0,0 +1,235 @@
+package pl.hycom.mokka.service.openapi;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import com.github.tomakehurst.wiremock.http.RequestMethod;
+import com.github.tomakehurst.wiremock.stubbing.StubMapping;
+import io.swagger.v3.oas.models.*;
+import io.swagger.v3.oas.models.media.Content;
+import io.swagger.v3.oas.models.media.MediaType;
+import io.swagger.v3.oas.models.media.Schema;
+import io.swagger.v3.oas.models.responses.ApiResponse;
+import io.swagger.v3.oas.models.responses.ApiResponses;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import pl.hycom.mokka.service.openapi.pojo.StubMappingListWrapper;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author Kamil Adamiec (kamil.adamiec@hycom.pl)
+ */
+public class OpenAPIToStubMappingListWrapperConverterUnitTest {
+
+    private static final String TEST_PATH = "/path";
+
+    private static final String HTTP_STATUS_OK_CODE = "200";
+
+    private static final String HTTP_STATUS_NOT_FOUND_CODE = "404";
+
+    private static final String CONTENT_TYPE_HEADER_KEY = "Content-Type";
+
+    private static final String APPLICATION_JSON_CONTENT_TYPE = "application/json";
+
+    private static final String APPLICATION_XML_CONTENT_TYPE = "application/xml";
+
+    private static final String RESPONSE_AS_XML = "<version>3.0.0</version>";
+
+    private static final String RESPONSE_AS_JSON = "\"{\\\"version\\\":\\\"3.0.0\\\"}\"";
+
+    @Mock
+    private OpenApiSchemaParsingStrategy openApiSchemaParsingStrategy;
+
+    @InjectMocks
+    private OpenAPIToStubMappingListWrapperConverter testedConverter = new OpenAPIToStubMappingListWrapperConverter();
+
+    @BeforeEach
+    public void setUp() {
+
+        MockitoAnnotations.initMocks(this);
```

Comment:
not needed when `@InjectMocks` is used. On class level you should add: `@ExtendWith(MockitoExtension.class)`